example/test.go

```diff
@@ -0,0 +1,17 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+)
+
+func main() {
```

Comment:
Do we need this?

Reply:
Woah, thanks for catching this. It was just a quick test script. I will delete it.